### PR TITLE
Revise precision range and description for geohash and geotile grid

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Revise precision range and description for geohash and geotile grid

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _aggregations/bucket/geohash-grid.md [hunk 1]
```diff
@@ -255,7 +255,7 @@ Geohash grid aggregation requests support the following parameters.
 Parameter | Data type | Description
 :--- | :--- | :---
 field | String | The field on which aggregation is performed. This field must be mapped as a `geo_point` or `geo_shape` field. If the field contains an array, all array values are aggregated. Required.
-precision | Integer | The zoom level used to determine grid cells for bucketing results. Valid values are in the [0, 15] range. Optional. Default is 5. 
+precision | Integer | The granularity level used to determine grid cells for bucketing results. Valid values are in the [0, 12] range. Optional. Default is 5.
```
Comment (natebower) [inline _aggregations/bucket/geohash-grid.md:258]: Should we say what about the grid cells is being determined?

**Review by natebower — APPROVED** [review]
@kolchfa-aws Just one comment. Otherwise, LGTM

File: _aggregations/bucket/geohash-grid.md (hunk 1 above)
Comment (kolchfa-aws) [inline _aggregations/bucket/geohash-grid.md:258]:
```suggestion
precision | Integer | The granularity level used to determine grid cells for bucketing results. Cells cannot exceed the specified size (diagonal) of the required precision. Valid values are in the [0, 12] range. Optional. Default is 5. 
```

File: _aggregations/bucket/geohex-grid.md [hunk 2]
```diff
@@ -387,7 +387,7 @@ Geohex grid aggregation requests support the following parameters.
 Parameter | Data type | Description
 :--- | :--- | :---
 field | String | The field that contains the geopoints. This field must be mapped as a `geo_point` field. If the field contains an array, all array values are aggregated. Required.
-precision | Integer | The zoom level used to determine grid cells for bucketing results. Valid values are in the [0, 15] range. Optional. Default is 5. 
+precision | Integer | The granularity level used to determine grid cells for bucketing results. Valid values are in the [0, 15] range. Optional. Default is 5.
```
Comment (kolchfa-aws) [inline _aggregations/bucket/geohex-grid.md:390]:
```suggestion
precision | Integer | The granularity level used to determine grid cells for bucketing results. Cells cannot exceed the specified size (diagonal) of the required precision. Valid values are in the [0, 15] range. Optional. Default is 5. 
```

File: _aggregations/bucket/geotile-grid.md [hunk 3]
```diff
@@ -544,7 +544,7 @@ Geotile grid aggregation requests support the following parameters.
 Parameter | Data type | Description
 :--- | :--- | :---
 field | String | The field that contains the geopoints. This field must be mapped as a `geo_point` field. If the field contains an array, all array values are aggregated. Required.
-precision | Integer | The zoom level used to determine grid cells for bucketing results. Valid values are in the [0, 15] range. Optional. Default is 5. 
+precision | Integer | The granularity level used to determine grid cells for bucketing results. Valid values are in the [0, 29] range. Optional. Default is 7.
```
Comment (kolchfa-aws) [inline _aggregations/bucket/geotile-grid.md:547]:
```suggestion
precision | Integer | The granularity level used to determine grid cells for bucketing results. Cells cannot exceed the specified size (diagonal) of the required precision. Valid values are in the [0, 29] range. Optional. Default is 7. 
```

File: _aggregations/bucket/geohash-grid.md (hunk 1 above)
Comment (kolchfa-aws) [reply _aggregations/bucket/geohash-grid.md]: Added more info.

**Pull request merged**